keyboards/ergodox_ez/keymaps/default/keymap.c

```diff
@@ -194,7 +194,9 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 
 // Runs just one time when the keyboard initializes.
 void matrix_init_user(void) {
-
+#ifdef RGBLIGHT_COLOR_|LAYER_0
```

Comment:
Looks like you have an extra `|` here :)

Reply:
Whoops!  Fixed